File: 360-video-player-for-android-master/app/src/main/java/com/oculus/sample/CustomAdapter.java
package com.oculus.sample;

import android.content.Context;
import android.support.annotation.NonNull;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.ArrayAdapter;
import android.widget.ImageView;

/**
 * Created by Prashant on 23-03-2017.
 */

public class CustomAdapter extends ArrayAdapter<String> {

    Integer[] pics;
    String[] url;
    Context context ;

    public CustomAdapter (Context context, Integer[] pics, String[] url){

        super(context, R.layout.customadapter,url);

        this.pics=pics;
        this.context = context;
        this.url= url;
    }

    @NonNull
    @Override
    public View getView(int position, View convertView, ViewGroup parent) {
        LayoutInflater layoutInflater = (LayoutInflater)context.getSystemService(Context.LAYOUT_INFLATER_SERVICE);

        convertView =layoutInflater.inflate(R.layout.customadapter,null);
        ImageView imageView =(ImageView)convertView.findViewById(R.id.imageView);
        imageView.setBackgroundResource(pics[position]);
        imageView.setScaleType(ImageView.ScaleType.FIT_XY);
        return convertView;
    }
}

File: 360-video-player-for-android-master/app/src/main/java/com/oculus/sample/CustomAdapter.kt
package com.oculus.sample

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.ArrayAdapter
import android.widget.ImageView

class CustomAdapter(
    context: Context,
    private val pics: Array<Int>,
    private val url: Array<String>
) : ArrayAdapter<String>(context, R.layout.customadapter, url) {

    override fun getView(position: Int, convertView: View?, parent: ViewGroup): View {
        var newConvertView = convertView
        val layoutInflater = context.getSystemService(Context.LAYOUT_INFLATER_SERVICE) as LayoutInflater
        if (newConvertView == null) {
            newConvertView = layoutInflater.inflate(R.layout.customadapter, null)
        }
        val imageView = newConvertView!!.findViewById<ImageView>(R.id.imageView)
        imageView.setBackgroundResource(pics[position])
        imageView.scaleType = ImageView.ScaleType.FIT_XY
        return newConvertView
    }
}
